src/components/Votes/ApprovalCastVoteButton/ApprovalCastVoteButton.tsx

```diff
@@ -77,6 +87,58 @@ export default function ApprovalCastVoteButton({
   return (
     <VStack className="flex-shrink-0">
       <VStack alignItems="items-stretch">
+        {isReady && (
+          <div className="pt-3">
+            <TooltipProvider>
+              <Tooltip>
+                <TooltipTrigger className="w-full flex items-center justify-center gap-1 text-primary font-medium cursor-help">
+                  <span className="flex items-center">
+                    Proposal voting power{"\u00A0"}
+                    <TokenAmountDisplay amount={votingPower.totalVP} />
+                    <Image
+                      src={icons.info}
```

Comment:
probably doesn't matter that much but the figma mockup looks like it's outline info icon, not solid. We don't have a cohesive strategy for icons, but we do use heroicons and you could grab the outline version with `import { InformationCircleIcon } from "@heroicons/react/24/outline"`